packages/components/src/menu/index.tsx

```diff
@@ -2,28 +2,18 @@
  * External dependencies
  */
 import * as Ariakit from '@ariakit/react';
-import { useStoreState } from '@ariakit/react';
 
 /**
  * WordPress dependencies
  */
-import {
-	useContext,
-	useMemo,
-	cloneElement,
-	isValidElement,
-	useCallback,
-} from '@wordpress/element';
-import { isRTL } from '@wordpress/i18n';
-import { chevronRightSmall } from '@wordpress/icons';
+import { useContext, useMemo } from '@wordpress/element';
+import { isRTL as isRTLFn } from '@wordpress/i18n';
```

Comment:
Why was the alias necessary?

Reply:
Not a "necessary" change, but I refactored the code in a way such that `isRTL` became a local variable, and therefore I renamed the import to avoid the naming conflict (search for `computedDirection` in the deleted code)